addons/zeusFactory/functions/fnc_ai_onRTB.sqf

```diff
@@ -0,0 +1,8 @@
+#include "script_component.hpp"
+
+params ["_leader"];
+private _vehicle = vehicle _leader;
+TRACE_2("ai_onRTB",_leader,_vehicle);
+
+deleteVehicle _leader;
+deleteVehicle _vehicle;
```

Comment:
Yeah, I saw it a while back. We should probably update to the new syntax across our mod when it goes live. I didn't recommend it here because I didn't want it to rely on 1.68